### PR TITLE
[PWA-1542] Verify Buttons

### PR DESCRIPTION
**Pull request by eug123**
## Description

To verify that the Page Builder Buttons content type renders correctly on Venia for each content type property.

## Related Issue

Closes [PWA-1542](https://jira.corp.magento.com/browse/PWA-1542): Verify Buttons

### Verification Steps

1. Cypress `buttons.spec.js` should pass
Steps - 
Go to venia-integration-test dir
Run `yarn install`
Run `yarn test`
Run `buttons.spec.js` from cypress ui

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: venia-integration-tests/src/tests/pageBuilder/integrationTests/buttons.spec.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+describe('verify pagebuilder buttons content is rendered correctly', () => {
+    it('verify buttons content', () => {
+        cy.intercept('GET', '**/graphql?query=query+GetCmsPage*', {
+            fixture: 'pageBuilder/buttons/buttons.json'
+        }).as('getCMSMockData');
+        cy.visit('/');
```
Comment (dpatil-magento) [inline venia-integration-tests/src/tests/pageBuilder/integrationTests/buttons.spec.js:6]: Use `visitHomePage` from  https://github.com/magento-obsessive-owls/pwa-studio/blob/PWA-1542/venia-integration-tests/src/support/routesCommands.js

Comment (sirugh) [reply venia-integration-tests/src/tests/pageBuilder/integrationTests/buttons.spec.js:6]:
```suggestion
        cy.visitHomePage()
```

Comment (eug123) [reply venia-integration-tests/src/tests/pageBuilder/integrationTests/buttons.spec.js]: Done, thank you

**Review by sirugh — APPROVED** [review]

**Review by dpatil-magento — APPROVED** [review]

**Comment by dpatil-magento** [comment]
QA Approved.

**Pull request merged**